test/clj/chronograph/middleware_test.clj

```diff
@@ -3,34 +3,67 @@
             [chronograph.factories :as factories]
             [chronograph.fixtures :as fixtures]
             [chronograph.middleware :as middleware]
-            [chronograph.utils.time :as time]
-            [clojure.test :refer :all])
-  (:import java.time.temporal.ChronoUnit))
+            [clojure.test :refer :all]
+            [ring.util.response :as response]))
 
-(use-fixtures :once fixtures/config fixtures/datasource)
+(use-fixtures :once fixtures/config fixtures/datasource fixtures/report-logging-only)
 (use-fixtures :each fixtures/clear-db)
 
-(deftest wrap-authenticated-middleware-when-user-authentic-test
-  (testing "Should return updated request having user's information, fetched from the DB, if the user is authentic."
-    (let [user (factories/create-user)]
+(deftest wrap-cookie-auth-test
+  (let [user (factories/create-user)]
+    (testing "Should return updated request having user's information, fetched from the DB, if the user is authentic."
       (is (= user
-             (:user ((middleware/wrap-authenticated-user identity)
+             (:user ((middleware/wrap-cookie-auth identity)
                      {:cookies {"auth-token" {:value (auth/create-token
-                                                      (:users/id user))}}})))))))
+                                                       (:users/id user))}}})))))
 
+    (testing "Should return updated request with nil user, if the user is NOT authentic."
+      (let [non-existent-user-id (+ (rand-int 100)
+                                    1
+                                    (:users/id user))]
```

Comment:
Perhaps we could just use `Long/MAX_VALUE` and be done with it? Arithmetic in tests = bugs in tests.

Long because autoincrement serial gets coerced to long (afaik), and `int?` checks for long integer also (from spec point of view).